extensions/positron-python/src/client/terminals/envCollectionActivation/service.ts

```diff
@@ -409,9 +421,18 @@ export class TerminalEnvVarCollectionService implements IExtensionActivationServ
               };
     }
 
-    private getEnvironmentVariableCollection(scope: EnvironmentVariableScope = {}) {
-        const envVarCollection = this.context.environmentVariableCollection as GlobalEnvironmentVariableCollection;
-        return envVarCollection.getScoped(scope);
+    // --- Start Positron ---
+    // Global Environment Variable Collections will overwrite other additions to the same environment variable
+    // from other extensions, eg, Quarto
+
+    private getEnvironmentVariableCollection() {
+        // private getEnvironmentVariableCollection(scope: EnvironmentVariableScope = {}) {
+        //     const envVarCollection = this.context.environmentVariableCollection as GlobalEnvironmentVariableCollection;
+        //     return envVarCollection.getScoped(scope);
+
+        const envVarCollection = this.context.environmentVariableCollection as EnvironmentVariableCollection;
```

Comment:
IIUC this line change isn't necessary. This tells the TS compiler that the object is an `EnvironmentVariableCollection` but doesn't affect anything at runtime. The object is probably actually a `GlobalEnvironmentVariableCollection`. The main runtime change here is returning the object as is instead of the result of `getScoped`.

Reply:
> The main runtime change here is returning the object as is instead of the result of getScoped.

That is correct! I was getting type errors in the tests on build/compile without explicitly specifying, which is why I made this change. Since the `getScoped` isn't being used, I think it is benign?